pkg/icingadb/sla.go

```diff
@@ -0,0 +1,137 @@
+package icingadb
+
+import (
+	"context"
+	"github.com/icinga/icingadb/pkg/contracts"
+	v1 "github.com/icinga/icingadb/pkg/icingadb/v1"
+	"github.com/icinga/icingadb/pkg/types"
+	"github.com/pkg/errors"
+	"golang.org/x/sync/errgroup"
+	"time"
+)
+
+type SlaHistoryTrail struct {
+	Id                 types.Int `json:"id" db:"-"`
```

Comment:
`db:"-"` is correct. But its reason isn’t obvious for a not involved programmer.